doc/api/assert.markdown

```diff
@@ -2,42 +2,131 @@
 
     Stability: 3 - Locked
 
-This module is used so that Node.js can test itself. It can be accessed with
-`require('assert')`. However, it is recommended that a userland assertion
-library be used instead.
+The `assert` module was originally written and included in Node.js in order
+to facilitate Node.js' own unit tests. While it can be used by JavaScript
+running within Node.js by using `require('assert')`, it is recommended that a
+userland assertion module be used instead.
+
+The API for the `assert` module is considered to be "locked". This means that
+there will be no additions or changes to any of the methods currently
+implemented and exposed by the module.
```

Comment:
Maybe just include a link to the explanation of Locked that exists elsewhere in the docs.